go.mod

```diff
@@ -1,9 +1,9 @@
 module istio.io/client-go
 
-go 1.12
+go 1.15
```

Comment:
Users will not be broken if they use a different version. This is only a suggestion according to the documentation.

Reply:
Any concern if we use go 1.16?